powershell_shortcut-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,14 @@
+package:
+  name: powershell_shortcut
+  version: 0.0.1
+
+build:
+  number: 3
```

Comment:
probably better to start this out at 0?

Reply:
Well, version 0.0.0 looks weird, isn't it? "but build: number: 3" I was not sure for what it actual stands. So shall I use here 0 instead?

Reply:
Yep 0 is better is start a recipe with.  Version number corresponds to source code (which in this case, really is the recipe, so version number is all we should ever bump).  Build number is when you have the same base source code, but you tweak the build scripts or add a patch since the last time a package was built.  Ideally, you always reset the build number to 0 when you increase the version number.